src/DeviceInterface/AndroidInterface.js

```diff
@@ -40,9 +40,44 @@ class AndroidInterface extends InterfacePrototype {
         return Promise.resolve(userData)
     }
 
+    getEmailProtectionCapabilities () {
```

Comment:
I think we should mark with a comment which of these calls are only available on our web page. This is one, I think.